modules/public/static.go

```diff
@@ -20,6 +20,15 @@ import (
 	"code.gitea.io/gitea/modules/log"
 )
 
+func GlobalModTime(filename string) time.Time {
+	t, err := util.GetExecutableModTime()
```

Comment:
Note: `GetExecutableModTime` isn't a function, and is also a reason why CI fails.